### DISCUSSION TITLE
Proper Documentation request for Multitenancy feature

### DISCUSSION THREAD

**Discussion by vedraiyani** [adrien2p/medusa-extender#90] [Q&A] [answered]
I followed the doc for multitenancy where it's mentioned to set up the config but nothing is said about how we can configure the tenant for the store. I could also see that `Multi-tenant 1` is mentioned so how to configure DB for each tenant?

**Comment by adrien2p** [comment, marked as answer]
hey 👋 it has been release as part of the new version (1.7.2)
